proposals/partial-events-and-constructors.md

```diff
@@ -185,9 +185,9 @@ A defining partial constructor declaration cannot have a constructor initializer
 ### Attributes
 
 The attributes of the resulting event or constructor are the combined attributes of the partial declarations.
-The attributes of each of the resulting event's accessor
-are the combined attributes of the corresponding partial declaration accessors.
-The attributes of each of the resulting constructor's parameter
+The attributes of each of the resulting event accessors
+are the combined attributes of the corresponding partial implementing declaration accessors.
```

Comment:
Since there should be at most one implementation, perhaps:
"... are the ~~combined~~ attributes of the corresponding partial implementing declaration accessor~~s~~."